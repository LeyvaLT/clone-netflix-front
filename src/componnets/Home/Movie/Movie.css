.card_movie{
    width: 13rem;
}
.img_movie{
    padding: 0.4rem;
}