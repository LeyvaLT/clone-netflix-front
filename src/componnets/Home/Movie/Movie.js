import React, {Component} from 'react';
import './Movie.css';

class Movie extends Component{

    render(){
        return(
            <div className="col-md">
            <div className="card card_movie">
                <img className="card-img-top img_movie" src="https://vignette.wikia.nocookie.net/doblaje/images/d/df/Suits-USA-season-2-2012-poster.jpg/revision/latest?cb=20120703033409&path-prefix=es" alt="Card image cap"/>
                    <div className="card-body">
                        <h5 className="card-title"><strong>SUITS</strong></h5>
                        <p className="card-text"></p>
                        <a href="#" className="btn btn-primary">Ver</a>
                    </div>
            </div>
            </div>
                )
    }

}

export default Movie;