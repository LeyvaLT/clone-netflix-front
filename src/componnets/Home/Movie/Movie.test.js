import React from 'react';
import {shallow} from 'enzyme';
import Movie from './Movie';


describe("Movie", () => {

    const movieObject = {
        "poster": "www.google.com",
        "title": "La pelicula Test",
        "id": "NWEKSDNJWHEOIW"
    };

    const  component = shallow (<Movie title={movieObject.title}
                                       poster={movieObject.poster}
                                       id={movieObject.id}
    />);

    it ("Check it props passed correctly", () => {
        expect(component.find(".card-title").text()).toBe(movieObject.title);
    });

    it("Check movie renders fine", () => {
        expect(component).toMatchSnapshot();
    });

});
