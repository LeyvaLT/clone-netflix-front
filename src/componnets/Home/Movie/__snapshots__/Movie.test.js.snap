// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie Check movie renders fine 1`] = `
<div
  className="col-md d-flex align-items-stretch"
>
  <div
    className="card"
  >
    <img
      alt="Card image cap"
      className="card-img-top img_movie"
      src="www.google.com"
    />
    <div
      className="card-body"
    >
      <h5
        className="card-title"
      >
        <strong>
          La pelicula Test
        </strong>
      </h5>
      <Link
        className="btn btn-primary"
        replace={false}
        to="/movie/NWEKSDNJWHEOIW"
      >
        Ver
      </Link>
    </div>
  </div>
</div>
`;
